packages/perseus/package.json

```diff
@@ -61,7 +61,7 @@
         "@khanacademy/wonder-blocks-layout": "2.0.32",
         "@khanacademy/wonder-blocks-link": "6.1.1",
         "@khanacademy/wonder-blocks-pill": "2.2.1",
-        "@khanacademy/wonder-blocks-popover": "3.2.2",
+        "@khanacademy/wonder-blocks-popover": "3.2.9",
```

Comment:
**todo**: our WB dependencies in Perseus are a bit annoying to manage.  We need to keep the `devDependencies` in sync with the `peerDependencies` later in the same file.

The `devDependencies` are what our build/test/storybook tooling look at, but the `peerDependencies` help to notify consuming apps (webapp) what our base level of the dependency is.